lib/team_api/joiner.rb

```diff
@@ -20,6 +20,7 @@ class Joiner
     def self.join_data(site)
       impl = JoinerImpl.new site
       site.data.merge! impl.collection_data
+      impl.create_indexes
```

Comment:
since we need to do this here so the private data doesn't get filtered, do you think it makes sense to add a test for that so someone doesn't come in and rearrange things in a way that would break this new improved behavior in the future?